sql/core/src/test/scala/org/apache/spark/sql/DataFrameSetOperationsSuite.scala

```diff
@@ -428,7 +428,7 @@ class DataFrameSetOperationsSuite extends QueryTest with SharedSparkSession {
     errMsg = intercept[AnalysisException] {
       df1.unionByName(df2)
     }.getMessage
-    assert(errMsg.contains("""Cannot resolve column name "b" among (a, c, d)"""))
+    assert(errMsg.contains("""Cannot resolve column name `b` among (a, c, d)"""))
```

Comment:
Shall we keep the previous error message? In general we should only use backticks when necessary(has special chars)

Reply:
ok. let me revert to previous one.